tests/test_api_model.py

```diff
@@ -319,6 +320,95 @@ def test_model_jacobian(
                 assert pytest.approx(JTf_inertial) == JTf_other, vel_repr.name
 
 
+def test_coriolis_matrix(
+    jaxsim_models_types: js.model.JaxSimModel,
+    velocity_representation: VelRepr,
+    prng_key: jax.Array,
+):
+
+    model = jaxsim_models_types
+
+    key, subkey = jax.random.split(prng_key, num=2)
```

Comment:
Unused variable
```suggestion
    _, subkey = jax.random.split(prng_key, num=2)
```

Reply:
Done in [31bbed9](https://github.com/ami-iit/jaxsim/pull/172/commits/31bbed9c9085ae89ec6635932b6569d88bab5994)